### PR TITLE
[JRO] High level ideas + requirements for the app

### PR DESCRIPTION
**Pull request by jayroh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,44 @@
-# rubocaucus
-Let your team vote on rubocop style enforcement
+# Rubocaucus
+
+## Name?
+
+1. rubocaucus
+2. rubolection
+3. rubocommissioner
+4. ruboconsensus
```
Comment (TheDudeWithTheThing) [inline README.md]: rubosensus ?

Comment (jayroh) [inline README.md]: Another possibility! I'll add it in

File: README.md [hunk 2]
```diff
@@ -1,2 +1,44 @@
...
+In order to align a team of ruby developers for the purposes of the best software *they* can write,
+a shared understanding and agreement of the rules of the project must be come upon together and
+enforced with the rubocop tool. Rubocaucus is the tool to help reach consensus and move forward as
+one unit.
+
+## Requirements
+
+As a team leader/admin I want to:
+
+1. Create a team profile
```
Comment (TheDudeWithTheThing) [inline README.md]: We can probably use github for login / team enforcement

**Comment by TheDudeWithTheThing** [comment]
This is a good start, we can do more details somewhere else, or get a skeleton project going and create the routes file to discuss resources / endpoints / etc...

**Comment by jayroh** [comment]
Yup! That's a great idea.

Rails 5?

**Comment by TheDudeWithTheThing** [comment]
I was thinking rails 5 but maybe we upgrade to that once we have a real client / server model going and can `--api-only` it

**Comment by jayroh** [comment]
`--api-only` would then make this a rails + ember/angular/react app, then. No? I think adding that sort of dependency on top of it would add another barrier to contributing.

**Comment by TheDudeWithTheThing** [comment]
that's pretty much the only feature I know that's new in Rails 5, if there are other benefits I'm all aboard!

**Comment by jayroh** [comment]
There's some other good stuff in there too, attributes API, etc. We probably won't use much of it, if any, at all but it's worthwhile being up to date and ahead of the curve.

**Pull request merged**